index.js

```diff
@@ -131,18 +131,18 @@ function createMedicationResponseCard(context) {
           indicator: 'info',
           source: {
             label: 'CDS Service Tutorial',
-            url: 'https://github.com/cerner/cds-services-tutorial/wiki/Medication-Prescribe-Service'
+            url: 'https://github.com/cerner/cds-services-tutorial/wiki/Order-Select-Service'
```

Comment:
This wiki doesn't work yet.

Reply:
Correct.  I was looking to add the code content first